docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,69 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
+object keys be placed on separate lines for better readability.
+
+## Rule Details
+
+This rule aims to maintain consistency of newlines between object keys.
+
+Examples of **correct** code for this rule:
+
+```js
+/*eslint object-key-newlines: "error"*/
+
+var obj = {
+    foo: "foo",
+    bar: "bar",
+    baz: "baz"
+};
+```
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint object-key-newlines: "error"*/
+
+var obj = { foo: "foo", bar: "bar", baz: "baz" };
+
+var obj2 = {
+    foo: "foo", bar: "bar", baz: "baz"
+};
+
+var obj3 = {
+    foo: "foo", bar: "bar",
+    baz: "baz"
+};
+```
+
+## Options
+
+This rule has an object option:
+
+* `"allowMultipleKeysPerLine"`: `true` allows all keys and values to be on the same line
+
+Examples of additional **correct** code for this rule with the `{ "allowMultipleKeysPerLine": true }` option:
```

Comment:
Yes, good catch! ### allowMultipleKeysPerLine after the bulleted list item and before the Examples sentence.